src/main/webapp/app/entities/participation/participation-websocket.service.ts

```diff
@@ -173,20 +184,18 @@ export class ParticipationWebsocketService {
      */
     private addResultToParticipation(result: Result) {
         const correspondingParticipation = this.cachedParticipations.get(result.participation.id);
-        if (!correspondingParticipation.results) {
-            correspondingParticipation.results = [];
-        }
-        correspondingParticipation.results = correspondingParticipation.results.map(el => (el.id === result.id ? result : el));
-        if (!correspondingParticipation.results.some(el => el.id === result.id)) {
-            correspondingParticipation.results.push(result);
-        }
-        this.cachedParticipations.set(correspondingParticipation.id, correspondingParticipation);
-        if (this.participationObservable) {
-            this.participationObservable.next(correspondingParticipation);
-        }
-        const resultObservable = this.resultObservables.get(correspondingParticipation.id);
-        if (resultObservable) {
-            resultObservable.next(result);
+        if (correspondingParticipation) {
```

Comment:
The only thing I have changed here is that the corresponding participation is checked.
If no participation was yet added to the service, the caching can not take place.